scripts/generate_crds_manifests.sh

```diff
@@ -495,3 +492,54 @@ find "${ROOT_DIR}/manifests" -type f -exec $SED -i "1{/---/d}" {} \;
 # namespaces enforce restricted PSA policies, so warnings and audits labels are not neccessary.
 ${YQ} delete --inplace -d'*' manifests/0000_50_olm_00-namespace.yaml 'metadata.labels."pod-security.kubernetes.io/warn*"'
 ${YQ} delete --inplace -d'*' manifests/0000_50_olm_00-namespace.yaml 'metadata.labels."pod-security.kubernetes.io/audit*"'
+
+
+# Let's copy all the manifests to a separate directory for microshift
+mkdir -p "${ROOT_DIR}/microshift-manifests/"
+rm -rf "${ROOT_DIR}/microshift-manifests/"*
+cp "${ROOT_DIR}"/manifests/* "${ROOT_DIR}/microshift-manifests/"
+
+# Let's add the annotations for ibm-cloud-managed on the standard manifests
+# !! Order matters, as we don't need the IBM managed annotations on the microshift manifests.
```

Comment:
What would happen if we did have these annotations on the manifests used by MicroShift? I'm thinking the less we differ, the better?

Reply:
done, PTAL